app/models/organization.rb

```diff
@@ -0,0 +1,7 @@
+class Organization < ActiveRecord::Base
```

Comment:
Make sure to use the `friendly_id` stuff for this based on the name of the organization

Make sure organization names are required and uniq as well